docs/content/getting_started/index.md

```diff
@@ -182,21 +182,26 @@ git config --global user.email "username@domain.com"
 git config --global user.name "username"
 ```
 
+### JDK + Maven (If using Type Safe API)
+
+The Type Safe API requires JDK >= 11 and Maven >= 3.8 in order to perform the required code generation.
+
+To install these, follow the below links:
+
+- JDK: We recommend installing any Coretto version >= 11 https://docs.aws.amazon.com/corretto/
+- Maven: You can any version of Maven >= 3.8: https://maven.apache.org/install.html
+
 ### Language specific
 
 Other prerequisites depend on the language in which you develop AWS PDK projects and are as follows.
 
 === "TYPESCRIPT"
-    - `Node >= 16`
-    - `PNPM >= 8.6.3` [if using `--package-manager=pnpm` flag to bootstrap]
+    - `PNPM >= 8.6.3` [if using `--package-manager=pnpm` flag to bootstrap]: https://pnpm.io/installation
+    - `YARN` [if `--package-manager` flag is unset]: https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable
```

Comment:
I think Java and Python also need yarn so that any ts subprojects can depend on one another? :)

Reply:
Fixed :)